CSLabs.Api/Controllers/SystemMessagesController.cs

```diff
@@ -0,0 +1,51 @@
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using CSLabs.Api.Models;
+using CSLabs.Api.Models.ModuleModels;
+using CSLabs.Api.RequestModels;
+using CSLabs.Api.Util;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace CSLabs.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    [Authorize]
+
+    public class SystemMessagesController : BaseController
+    {
+        public SystemMessagesController(BaseControllerDependencies dependencies) : base(dependencies)
+        {
+
+
+            [HttpGet("{id}")] 
+            public async Task<IActionResult> GetSystemMessageID(int id)
```

Comment:
I'm not sure we're ever going to need to directly get a single message, for the most part we will get all the messages for our timeframe or all the messages to show in a list.